src/js/game/hud/parts/building_placer_logic.js

```diff
@@ -194,12 +219,12 @@ export class HUDBuildingPlacerLogic extends BaseHUDPart {
         const selectedBuilding = this.currentMetaBuilding.get();
         if (selectedBuilding) {
             if (this.root.keyMapper.getBinding(KEYMAPPINGS.placement.rotateInverseModifier).pressed) {
-                this.currentBaseRotation = (this.currentBaseRotation + 270) % 360;
+                this.setBaseRotation((this.getBaseRotation() + 270) % 360);
```

Comment:
If you change the code into `get currentBaseRotation() {}` you don't have to touch the remaining code. Can also use setters there

Reply:
Yeah, I wasn't very familiar with this syntax in JS. I made the switch now.